test/blackbox/common/mesh_event_handler.h

```diff
@@ -90,6 +90,20 @@ struct cond_flag {
 	bool flag;
 };
 
+typedef struct {
+	mesh_event_t  currentEv;
+	mesh_event_t  nextEv;
+} meshlink_node_status_lookup_t;
+
+typedef struct {
+	uint16_t                        node_id;
+	mesh_event_t                    currentEv;
```

Comment:
Why do you have both `currentEv` and `current_index` in this struct? You always have `status->lookup[status->current_index].currentEv == status->currentEv`, so it is redundant. You should just keep `current_index`.

Reply:
Iam using currentEv to check if the current status of the node has reached a particular state or not. Functions like signal_node_start will use this variable to check if all nodes have joined and then send signals to all the nodes. Where as current_index is only used to indicate the current index of the next Event to occur.